Test/server/simple-session.transcript

```diff
@@ -1,632 +0,0 @@
-# RUN: %server "%s" > "%t"
```

Comment:
Because that test has gone stale with these changes -- there is a separate Issue logged to fix the documentation on how to repair such tests and to repair this one.